Source/Urho3D/Core/Profiler.h

```diff
@@ -37,8 +37,9 @@ void SetProfilerThreadName(const char* name);
 }
 
 #if URHO3D_PROFILING
-#   define URHO3D_PROFILE_C(name, color)          ZoneScopedNC(name, color)
-#   define URHO3D_PROFILE(name)                   ZoneScopedN(name)
+#   define URHO3D_PROFILE_FUNCTION()              ZoneScopedN(__FUNCTION__)
```

Comment:
Please also add such macro after `#else` so that build does not break when profiling is disabled.